rover/Profile.proto

```diff
@@ -0,0 +1,84 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+syntax = "proto3";
+
+package skywalking.v3;
+
+option java_multiple_files = true;
+option java_package = "org.apache.skywalking.apm.network.rover.profile.v3";
+option go_package = "skywalking.apache.org/repo/goapi/collect/rover/profile/v3";
+
+import "common/Common.proto";
+
+// Define the Rover Process profile task and upload profile data.
+service ProfileService {
+    // Query profile tasks
+    rpc queryTasks (ProfileTaskQuery) returns (Commands) {
+    }
+
+    // collect profile data
+    rpc collectProfileData (stream ProfileData) returns (Commands) {
+    }
+
+    // report profile task finished
+    rpc reportTaskFinish (ProfileTaskFinishReport) returns (Commands) {
+    }
```

Comment:
Do we need to skip this? As we consider the auto-profiling already?

Reply:
Nop, It's just a notification mechanism after the rover finished the profile task, then the OAP could know when the profile task has finished. If we don't need I could just delete it.

Reply:
My question is, with auto profiling, we can define what is an end. Because it starts and ends randomly and automatically.

Reply:
Updated. Delete the task finish request because we can't know when the task is finished. So we use the `profileStartTime` and `currentTime` in `ProcessProfileMetadata` to let the OAP save when it starts and ends